src/dist/replication/meta_server/meta_backup_service.cpp

```diff
@@ -1204,6 +1204,7 @@ void backup_service::start()
     start_create_policy_meta_root(after_create_policy_meta_root);
 }
 
+// TODO(zhaoliwei) refactor
```

Comment:
refactor what?

Reply:
I guess that liwei will refactor function `add_backup_policy`, its param is `message_ex *`

Reply:
I want to format this function, because it is too long and a little messy